<launch>
    <node pkg="first_project" type="pub_sub" name="odom_node"/>
    <param name="use_sim_time" value="true"/>

    <param name="starting_x" value="0.0"/>
    <param name="starting_y" value="0.0"/>
    <param name="starting_th" value="0.0"/>

    <node pkg="first_project" type="static_transform_publisher" name="front_left" value="1.85 0.93 0.0 0.81 0.0 3.14 vehicle_centre sick_front_left"/>
    <node pkg="first_project" type="static_transform_publisher" name="front_right" value="1.85 -0.93 0.0 -0.76 0.0 3.14 vehicle_centre sick_front_right"/>
    <node pkg="first_project" type="static_transform_publisher" name="rear_left" value="-1.85 0.93 0.0 2.38 0.0 3.14 vehicle_centre sick_rear_left"/>
    <node pkg="first_project" type="static_transform_publisher" name="rear_right" value="-1.75 -0.8 0.0 -2.30 0.0 3.14 vehicle_centre sick_rear_right"/>

</launch>